mlflow/models/wheeled_model.py

```diff
@@ -25,6 +25,7 @@
 _WHEELS_FOLDER_NAME = "wheels"
 _ORIGINAL_REQ_FILE_NAME = "original_requirements.txt"
 _PLATFORM = "platform"
+_MLFLOW_WHEELED_MODEL_PIP_DOWNLOAD_OPTIONS = "MLFLOW_WHEELED_MODEL_PIP_DOWNLOAD_OPTIONS"
```

Comment:
Could you define this here ? 
https://github.com/mlflow/mlflow/blob/master/mlflow/environment_variables.py and add doc for the defined `_EnvironmentVariable`


like:
```
# Add docs here ....
MLFLOW_WHEELED_MODEL_PIP_DOWNLOAD_OPTIONS = _EnvironmentVariable("MLFLOW_WHEELED_MODEL_PIP_DOWNLOAD_OPTIONS", str, "--only-binary=:all:")
```
Then we can get the env variable value by API `MLFLOW_WHEELED_MODEL_PIP_DOWNLOAD_OPTIONS.get()`